README.md

```diff
@@ -62,28 +62,38 @@ For more information, see the [.NET Foundation Code of Conduct](https://dotnetfo
 ## Examples
 
 Here's an example of code to train a model to predict sentiment from text samples. 
-(You can see the complete sample [here](test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs)):
+(You can find a sample of the legacy API [here](test/Microsoft.ML.Tests/Scenarios/SentimentPredictionTests.cs)):
```

Comment:
It is a bit confusing to point to the legacy sentiment sample and then have code for a different task (regression).